CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ------
 
+- (`#102 <https://github.com/znicholls/silicone/pull/102>`_) Minor bugfix for Equal Quantile Walk.
```

Comment:
I watched https://www.youtube.com/watch?v=NSemlYagjIU yesterday, could we have a slightly more descriptive message? Eg hot fixed equal quantile walk nan handling

Reply:
I've changed it, don't cry :)